CMakeLists.txt

```diff
@@ -142,6 +138,7 @@ endif()
 
 target_link_libraries(c_logging ${c_logging_target_libs})
 
+# run_traceability is an option from c build tools
```

Comment:
>run_traceability is an option from c build tools [](start = 2, length = 48)

I have a PR in c-build-tools to always create the target instead of this fix